setup/setup_pyrobosim.bash

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Sets up the pyrobosim virtual environment
+
+# User variables
+# Please modify these for your environment
+VIRTUALENV_FOLDER=~/python-virtualenvs/pyrobosim
+ROS_WORKSPACE=~/pyrobosim_ws
```

Comment:
You could potentially ask if the user wants to change them? 

And maybe make sure that both directories exist as part of this setup.

Reply:
I don't feel inclined to add even more prompts, so I think your suggestions on checking existence and erroring otherwise are good with me.